### ISSUE TITLE
Implementation details

### ISSUE THREAD

**Issue by yipingji** [ByungKwanLee/Causal-Unsupervised-Segmentation#1]
Hi,

Thanks for your great work!

I was just wondering that  I follow your instructions using my pretrained DINO encoder and the accuracy is very low (mIou ~5%).

**Comment by ByungKwanLee** [comment]
We train our model based on the baseline model, so if you change the baseline model with your pretrained model, it will not work.

**Comment by smeenapadnekar** [comment]
Please release the training code so that results are reproducible.
